assets/src/scss/new-identity/style.scss

```diff
@@ -123,9 +123,13 @@ table.spreadsheet-table.is-color-gp-green {
 
 a:where(.standalone-link) {
   width: fit-content;
-  border-bottom: 2px solid var(--gp-green-500);
   font-weight: 700 !important;
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+}
+
+a:where(.standalone-link, .external-link, .pdf-link),
+a:not(.single-post-author a, .tag-item, .comment-link, .site-logo, .nav-link, .btn, .article-list-item-headline a, .article-list-item-author a, .article-list-item-image a) {
```

Comment:
Looks good but honestly I see it very complex to maintain it afterwards.

But If this is the only way to lead with this scenario, then we can proceed to merge it and then think about a good solution for standardizing all the links.

Looking forward what @mleray thinks?

Reply:
I think that we should just use the `a` selector without all the classes, and fix the few links that have different styles such as the contextual links, because having such a complicated selector is really not ideal... It's harder to understand, and difficult to maintain for us and to override for NROs!